### DISCUSSION TITLE
Ph values from rem chem not fed to njs?

### DISCUSSION THREAD

**Discussion by lockeandev** [tagyoureit/nodejs-poolController#914] [Q&A]
![Screenshot_20240313_004803_Chrome](https://github.com/tagyoureit/nodejs-poolController/assets/11232490/a891a7ee-adae-4150-9820-ada8d458a47e)
Starting my pool up again and I noticed my ph values in dashPanel do not match the readings in rem chem. Attached are some screenshot. I'm not sure how long the feed has been broken or why. Any thoughts?
![Screenshot_20240313_004411_Chrome](https://github.com/tagyoureit/nodejs-poolController/assets/11232490/ba719e2a-1a21-4ec3-a361-e94ebf2131c6)
![Screenshot_20240313_004348_Chrome](https://github.com/tagyoureit/nodejs-poolController/assets/11232490/90c55b70-b4fd-4a65-8d0d-81adb2d3ff88)
![Screenshot_20240313_004333_Chrome](https://github.com/tagyoureit/nodejs-poolController/assets/11232490/5dccc21a-6c42-414c-9629-3b6f630705a9)

**Comment by tagyoureit** [comment]
Your screenshots you have “flow reading only” checked which means we won’t send the readings unless your pool Circuit is on.  So we would not expect new readings until you turn the pool on.

**Reply by lockeandev** [reply]
Good point. Looks like my flow switch is not working. I'll have the check if something is obstructing it or the connector fell off or something. When I turn off "flow ready only" the values come through fine. Thanks!

**Reply by tagyoureit** [reply]
Did your turn on the pool circuit?

**Reply by lockeandev** [reply]
Yes the pool circuit is on now, but ph was still not coming through until I unchecked the box because it shows that no flow is detected. But my pump is showing flow.
